dist/src/conf/dogma.json

```diff
@@ -28,14 +28,18 @@
   },
   "webAppEnabled": true,
   "webAppTitle": null,
-  "mirroringEnabled": null,
+  "mirroringEnabled": true,
   "numMirroringThreads": null,
   "maxNumFilesPerMirror": null,
   "maxNumBytesPerMirror": null,
   "replication": {
     "method": "NONE"
   },
   "csrfTokenRequiredForThrift": null,
```

Comment:
I see this field `csrfTokenRequiredForThrift ` in default config but not in documentation. Also the default configuration missing `cors` field

Reply:
Thanks. I added the `cors` field.